### PR TITLE
W-14023540 AEH  add instance info to API details page jh

### PR DESCRIPTION
**Pull request by JennyHajee**

### CODE REVIEW

**Review by spezzirriemiliano — DISMISSED** [review]

File: modules/ROOT/pages/viewing-api-details.adoc [hunk 1]
```diff
@@ -8,14 +8,37 @@ To view the API Details page, select an API from the Home or APIs page of the po
...
++
+The version of the API.
+* Instance
++
+The count of instances of the API.
 * Client applications
++
+The number of client applications of the API.
+
+The API Console is located in the API Details page. The console component is interactive and shows information about REST APIs and their methods. Endpoints are displayed in alphabetical order. See xref:api-console-specification-and-documentation.adoc[API Console Specification and Documentation] for more information.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/viewing-api-details.adoc]: not only rest api. We can put something like "API specifications"

Comment (JennyHajee) [reply modules/ROOT/pages/viewing-api-details.adoc]: Yes. I made this change.

File: modules/ROOT/pages/viewing-api-details.adoc [hunk 2]
```diff
@@ -8,14 +8,37 @@ To view the API Details page, select an API from the Home or APIs page of the po
...
++
+The count of instances of the API.
 * Client applications
++
+The number of client applications of the API.
+
+The API Console is located in the API Details page. The console component is interactive and shows information about API specifications and their methods. Endpoints are displayed in alphabetical order. See xref:api-console-specification-and-documentation.adoc[API Console Specification and Documentation] for more information.
+
+== View Instances Information and Copy the URL
+You can view the name and URL of an instance and copy the URL:
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/viewing-api-details.adoc]:
```suggestion
To view the name and URL of an instance and copy the URL:
```

File: modules/ROOT/pages/viewing-api-details.adoc [hunk 3]
```diff
@@ -8,14 +8,37 @@ To view the API Details page, select an API from the Home or APIs page of the po
 The API Details page includes the following information about an API:
 
 * Name
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/viewing-api-details.adoc]: Maybe these could be bold for UI text?

**Review by glenn-rodgers-sf — DISMISSED** [review]

File: modules/ROOT/pages/viewing-api-details.adoc (hunk 3 above)
Comment (JennyHajee) [reply modules/ROOT/pages/viewing-api-details.adoc]: Yes, good idea. done

**Review by glenn-rodgers-sf — APPROVED** [review]

**Pull request merged**